.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: bbbevents gem release
+on:
+  pull_request:
+    types:
+      - closed
```

Comment:
I don't think that pushing the gem on every closed pull request makes sense, since we might want to collect a few PRs before making a release.
That would also require that every PR must increase the version number, since you either can't - or at least don't want to - republish new code with the same version number.

It would be better to have the trigger for the gem release be tagging the git repo.